framework/web/View.php

```diff
@@ -492,7 +492,7 @@ private function registerFile($type, $url, $options = [], $key = null)
 
         if (empty($depends)) {
             // register directly without AssetManager
-            if ($appendTimestamp && Url::isRelative($url) && ($timestamp = @filemtime(Yii::getAlias('@webroot/' . ltrim($url, '/'), false))) > 0) {
+            if ($appendTimestamp && Url::isRelative($url) && ($timestamp = @filemtime(Yii::getAlias('@webroot/' . ltrim(ltrim($url, Yii::getAlias('@web')), '/'), false))) > 0) {
```

Comment:
This is not how trim works. All trim functions remove **chars** specified in second argument, so this will not only remove leading `Yii::getAlias('@web')` but also any further chars that are included in `@web` alias. Example: https://3v4l.org/Rjvgi

Reply:
Please read carefully, we use `ltrm` https://www.php.net/manual/de/function.ltrim.php

Reply:
See https://3v4l.org/ARlHc As indicated by @rob006 all trim functions work that way.

Reply:
i replaced it with another solution. I also added a unit test to cover this scenario.